src/v6.rs

```diff
@@ -0,0 +1,167 @@
+//! The implementation for Version 1 UUIDs.
+//!
+//! Note that you need to enable the `v1` Cargo feature
```

Comment:
I think you meant to say `v6` features
```suggestion
//! Note that you need to enable the `v6` Cargo feature
```